src/locale/locales/en.json

```diff
@@ -31,6 +31,14 @@
     "value": "Last page",
     "comment": "tooltip and aria-label for the last page button. (tew 210929)"
   },
+  "SNTable.Pagination.RowsPerPageChange": {
+    "value": "Rows per page has changed. Now showing {0} rows on each page and moved to the first page.",
```

Comment:
This string won't work for localization. In a number of the languages we translate in, there are multiple plural forms. This repo doesn't have the pluralization jsons that other repos have yet, so we'll have to rework these strings. Something like:

`Rows per page has changed (Rows: {0}). Now showing the first page.` 
`Rows per page has changed {0}. Now showing the first page.`